src/main/java/com/beust/jcommander/DefaultUsageFormatter.java

```diff
@@ -67,7 +67,7 @@ public final void usage(String commandName, StringBuilder out, String indent) {
 
         if (description != null) {
             out.append(indent).append(description);
-            out.append("\n");
+            out.append(System.getProperty("line.separator"));
```

Comment:
`System.lineSeparator()` is a bit better than this

Reply:
...and still as unnecessary, as I already explained. ;-)

Can you actually reproduce the reported problem? On which OS? In which shell?

Reply:
@yeikel : `System.lineSeparator()` always returns the same `String`, while `System.getProperty("line.separator")` can be changed by `System.setProperty("line.separator")`, which allows you to test emulating a foreign system. (so I do not have to fire up my windows computer .)